pkgs/top-level/all-packages.nix

```diff
@@ -9018,6 +9018,10 @@ with pkgs;
     openvpn_24
     openvpn;
 
+  openvpn3 = callPackage ../tools/networking/openvpn3 {
+    inherit (python3Packages) docutils jinja2 dbus-python wrapPython pygobject3;
```

Comment:
Makes me think maybe this should've been a `buildPythonModule`, but that can come in a future PR if anyone needs it.

Reply:
`openvpn3-linux` seems to be a complex derivation that mixes C code and Python code. I don't think we can use generic Python builders for that. Or I don't know how.

Reply:
The package uses make. buildPythonModule is for python build tools.